### DISCUSSION TITLE
photoshop pen pressure

### DISCUSSION THREAD

**Discussion by adrianc420** [OpenTabletDriver/OpenTabletDriver#1095] [Q&A] [answered]
i play osu with this driver and im trying to get in to digital art on photoshop, i want to be able to use the pen pressure (on my wacom ctl 472) with this driver and not have to install the wacom drivers. is there any way i can use pen pressure with OpenTabletDriver

**Comment by X9VoiD** [comment, marked as answer]
Use [WindowsInk](https://github.com/X9VoiD/VoiDPlugins/wiki/WindowsInk) plugin
